base/linalg/dense.jl

```diff
@@ -343,31 +345,16 @@ function ^{T}(A::AbstractMatrix{T}, p::Real)
         retmat = retmat * full(powm(UpperTriangular(float.(A)), real(p - floor(p))))
         d = diag(A)
     else
-        S,Q,d = schur(complex(A))
+        S,Q,d = schur(complex(full(A)))
```

Comment:
It should be possible to compute the complex Schur form from the real Schur form, but I guess that can be left for another PR.